nav2_constrained_smoother/CMakeLists.txt

```diff
@@ -74,8 +81,7 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-ament_export_include_directories(include)
```

Comment:
This is strictly  required.

Reply:
![image](https://github.com/ros-planning/navigation2/assets/25047695/9d5e424c-8ab6-4577-9fec-46db2a0d7629)

Can you elaborate? I switched to the target based method which is recommended by @sloretz . 
`ament_export_include_directories` is under consideration for deprecation in https://github.com/ament/ament_cmake/issues/365